specification/containerservice/Fleet.Management/update/run.tsp

```diff
@@ -101,6 +101,12 @@ model NodeImageSelection {
   @visibility("create", "read")
   @doc("The node image upgrade type.")
   type: NodeImageSelectionType;
+
+  @added(Versions.v2024_05_02_preview)
+  @visibility("create", "read")
```

Comment:
quite sure it needs update too here.
we do allow PUT on existing updaterun before they are started.

we have several other fields that are missing update, and we should fix them in 0502preview

Reply:
so just remove the visibility. default is create update read.

Reply:
please add an example file that uses the Custom option

Reply:
updated the GetUpdateRun.json to use the custom option.